Bynder/Test/Bynder.Test.csproj

```diff
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsTestProject>true</IsTestProject>
-    <AssemblyVersion>2.0.0</AssemblyVersion>
-    <FileVersion>2.0.0</FileVersion>
+    <AssemblyVersion>3.1.100</AssemblyVersion>
+    <FileVersion>3.1.100</FileVersion>
```

Comment:
I've created this one as well a while ago, in which the other projects (apart from Test) are updated as well. So maybe a good idea to merge that first, or combine them? https://github.com/Bynder/bynder-c-sharp-sdk/pull/37/

Reply:
I just saw that I will let you merge that and I will update with the master then 😄

Reply:
Yeah I've noticed I didn't change `AssemblyVersion` and `FileVersion` there. But no idea how you can do that from Visual Studio, apart from directly editing the files. And if it's important for something.

Reply:
Yeah I was playing around with the yml file we had 2.0.0 previously which doesn't work with the setting up the env in the actions script.